I/ImageMagick/ImageMagick@6/build_tarballs.jl

```diff
@@ -17,14 +17,14 @@ script = raw"""
 cd $WORKSPACE/srcdir/ImageMagick6*/
 
 if [[ "${target}" == *-linux-gnu ]]; then
```

Comment:
This should be needed only on Intel architectures because there we use a super-duper old glibc:
```suggestion
if [[ "${target}" == *86*-linux-gnu ]]; then
```